### PR TITLE
Provide an option to not automatically send gzipped files

### PR DESCRIPTION
**Pull request by andyhu**
Hi, I would suggest that it should not automatically send `.gz` files with same name of the request. There might be an old version of the `.gz` file created by the user and accidentally sent as gzipped version. This is a somewhat opinionated implementation, and programmers has no way to turn it off without hacking it. At least it should be configurable.
Any way, in this PR I made it default to enable gzip option for compatibility. There should be no side effect for users don't provide the option so it will not break anything.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -40,6 +40,7 @@ function send(ctx, path, opts) {
   var index = opts.index;
   var maxage = opts.maxage || opts.maxAge || 0;
   var hidden = opts.hidden || false;
+  var gzip = opts.gzip || true;
```
Comment (tj) [inline index.js]: this will always be true

Comment (andyhu) [inline index.js]: yes, thanks for pointing out. I've just fixed it

**Comment by jonathanong** [comment]
can you add a test and squash?

**Comment by andyhu** [comment]
ok, tests added

**Comment by jonathanong** [comment]
78be346cbe35a7f1496ee20c6a535f7026753aa5

**Pull request closed without merging**